artworks/admin.py

```diff
@@ -1,3 +1,11 @@
 from django.contrib import admin
-
+from .models import Artworks, Categories, ArtCategories, ArtFavourites, ArtLikes
 # Register your models here.
+
```

Comment:
Keep consistency. You either remove the auto-generated comment (which I recommend) or you keep it right before your code and leave space on top of it.